package org.folio.processing.exceptions;

public class MappingException extends RuntimeException {

  public MappingException(Throwable cause) {
    super(cause);
  }
}
